src/KSPACE/msm.cpp

```diff
@@ -140,10 +140,6 @@ void MSM::init()
   if ((order < 4) || (order > 10) || (order%2 != 0))
     error->all(FLERR,"MSM order must be 4, 6, 8, or 10");
 
-  if (sizeof(FFT_SCALAR) != 8)
```

Comment:
Is this safe to remove? I thought there was some reason for the error check, but can't remember, would have to look at the code.

Reply:
Yes. Nothing inside the msm code *uses* FFT_SCALAR except this line so there is no connection to FFT single.

Perhaps initially the thought was that single precision could be used as well to speed up communication, but the MSM code has been all but abandoned. If somebody wants to work on this, it would first need to be updated to the latest state. Dave Hardy told me at a meeting a few years ago, that there are a couple new papers that describe how it can be made faster and more accurate.